.tool-versions

```diff
@@ -3,7 +3,7 @@ gitleaks 8.24.0
 jq 1.6
 nodejs 22.15.0
 pre-commit 3.6.0
-python 3.12.11
```

Comment:
This has been intentionally left at 3.12 as it breaks the proxygen CLI otherwise